Tests/ApolloTests/Cache/ReadWriteFromStoreTests.swift

```diff
@@ -414,7 +414,70 @@ class ReadWriteFromStoreTests: XCTestCase, CacheDependentTesting, StoreLoading {
 
     let query = MockQuery<GivenSelectionSet>()
 
-    loadFromStore(query: query) { result in
+    loadFromStore(operation: query) { result in
+      try XCTAssertSuccessResult(result) { graphQLResult in
+        XCTAssertEqual(graphQLResult.source, .cache)
+        XCTAssertNil(graphQLResult.errors)
+
+        let data = try XCTUnwrap(graphQLResult.data)
+        XCTAssertEqual(data.hero.name, "Artoo")
+      }
+    }
+  }
+
+  func test_updateCacheMutation_givenMutationOperation_updateNestedField_updatesObjectAtMutationRoot() throws {
```

Comment:
Are we missing a test using `MockLocalCacheMutationFromSubscription`? _Or did I miss it in this PR?_

Reply:
I chose not to make one because it doesn't have any new behavior that isn't tested in the one for mutation. We need a test that we are calling `CacheReference.rootCacheReference(for: CacheMutation.operationType).key` at the call site in `ApolloStore`. Once we have one test verifying that, I didn't feel the need to add a redundant test for subscriptions. It's overkill that provides next to no value in so far as actually verifying real behavior. If you disagree I can add one though.